src/browser/views.py

```diff
@@ -343,12 +377,16 @@ def repo_collaborators_add(request, repo_base, repo):
     '''
     username = request.user.get_username()
     collaborator_username = request.POST['collaborator_username']
+    db_privileges = request.POST.getlist('db_privileges')
```

Comment:
This and a few other functions in browser/views.py should have bits that reject requests if they aren't POSTs. django.views.decorators.http's @require_POST is probably good for us to start using.